src/main/resources/assets/chisel/lang/de_DE.lang

```diff
@@ -23,7 +23,7 @@ chisel.tooltip.blockname=%1$s (%2$s)
 tile.chisel.blockAluminum.name=Aluminiumblock
 tile.chisel.blockAluminum.caution.desc.1=Vorsicht
 tile.chisel.blockAluminum.crate.desc.1=Versandkiste
-tile.chisel.blockAluminum.thermal.desc.1=Warm
+tile.chisel.blockAluminum.thermal.desc.1=Glühend
```

Comment:
I really don't have much time to maintain translations of any more mods with big language files, unfortunately.